File: Data Structures & Algorithms/rotate-array/submission-0.go
// [1,2,3,4,5,6,7,_]  8
// [8, 1,2,3,4,5,6,7,] // 8
// [7,8, 1,2,3,4,5,6,] // 7

func rotate(nums []int, k int) {
	for i:=0; i<k;i++ {
		end := nums[len(nums)-1]
		for i:=len(nums)-2; i>=0; i-- {
			nums[i+1] = nums[i]
		}
		nums[0] = end
	}
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
# [2,10,10,30,30,30]
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        unique, cur = 0, 1
        
        while cur < len(nums):
            if nums[cur] != nums[unique]:
                unique +=1
                nums[unique] = nums[cur]
                print(unique, nums[cur])
            cur += 1
        return unique+1